css/80_app.css

```diff
@@ -5777,6 +5777,12 @@ li.hide + li.version .badge .tooltip .popover-arrow {
 body {
     scrollbar-width: 10px;
 }
+/* Firefox */
+@-moz-document url-prefix() {
```

Comment:
[This documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/@document) says `@document` is deprecated and the prefixed version no longer works in author-defined stylesheets. Can we count on it to work on Firefox going forward?

Reply:
I see, is the intention to avoid clobbering the `width` here with whatever WebKit would use as `thin` in its (currently experimental) implementation?

https://github.com/openstreetmap/iD/blob/d04c3cdb73bc4661232d453ff749208ca7aedcba/css/80_app.css#L5749-L5754

I suppose this specific `url-prefix` hack does still work, so it’s an improvement over the status quo.

Reply:
Yes, I think the current WebKit scrollbar looks better, especially with the rounded borders, but I couldn’t find any way to achieve the same on Firefox.